spec/ruby/fixtures/code/concurrent_require_fixture.rb

```diff
@@ -0,0 +1,4 @@
+object = ScratchPad.recorded
+thread = Thread.new { object.require(__FILE__) }
+thread.wakeup unless thread.stop?
```

Comment:
Should be `Thread.pass until thread.stop?` here.

Other than that it looks good :+1:

Reply:
Fixed in https://github.com/ruby/ruby/pull/7270